src/Library/demos/Calendar/code.rs

```diff
@@ -0,0 +1,26 @@
+use crate::workbench;
+use glib::clone;
+use gtk::glib;
+
+pub fn main() {
+    let calendar: gtk::Calendar = workbench::builder().object("calendar").unwrap();
+
+    calendar.connect_day_notify(clone!(@weak calendar => move |_| {
```

Comment:
```suggestion
    calendar.connect_day_notify(clone!(@weak calendar => move |_| {
```

The closure gives you `calendar` so you don't need the `clone!` here